sdk/keyvault/keyvault-certificates/src/lro/recover/operation.ts

```diff
@@ -16,6 +16,7 @@ export type RecoverDeletedCertificateState = PollOperationState<KeyVaultCertific
 
 /**
  * An interface representing the state of a delete certificate's poll operation
+ * @internal
```

Comment:
is this comment useful given the fact api-extractor is smart enough to look at real exports? AFAIK this tag won't get used for anything in our processes, so it seems weird we're doing so much work to add it.

Reply:
api-extractor is smart enough, but typedoc isn't yet. We actually use `@internal` and `@ignore` on any exports that aren't meant to be user-facing in the event hubs library because of how this affects docs.

Reply:
It's smart enough to know what of these exports are publicly exported and which ones are not, but I don't know if some of these marked internal things could be useful from the docs perspective. I figure it's better to satisfy eslint for now and review that during docs review

Reply:
So, good news: the API docs will throw warnings if anything exported is marked as internal. I'm cleaning this now.

Reply:
It should be clean now!